tests/conftest.py

```diff
@@ -2343,8 +2343,9 @@ def finalizer():
 
     # Checks OCP version
     ocp_version = get_running_ocp_version()
+    ocp_version = "stable" if ocp_version >= "4.7" else ocp_version
```

Comment:
I would rename this variable to logging_channel

Reply:
done